plugins/devtools/README.md

```diff
@@ -34,6 +34,19 @@ Lists the configuration being used by your current running Backstage instance.
 
 ![Example of Config tab](./docs/devtools-config-tab.png)
 
+### Scheduled Tasks
+
+Scheduled tasks can be viewed and triggered under the `Scheduled Tasks` tab. You first must add the list of plugins for scheduled tasks to your config:
+
+```yaml
```

Comment:
For consistency can we move this down to the configuration section towards the bottom and then you can link to that from here.